docs/content/preview/yugabyte-platform/manage-deployments/remove-nodes.md

```diff
@@ -16,13 +16,55 @@ menu:
 type: docs
 ---
 
+## Replace a live or unreachable node
+
+If a node (live or unreachable) needs to be replaced, do the following:
+
+1. Navigate to **Universes**, select your universe, and open the **Nodes** tab.
+1. Click the associated node **Actions > Replace Node**.
+
+    ![Replace Node Actions](/images/ee/replace-node.png)
+
+1. Click OK to confirm.
+
+YugabyteDB Anywhere (YBA) starts the node replacement process, and you can view the progress on the **Tasks** tab.
+
+For cloud providers (AWS, Azure, or GCP), YBA returns the node back to the provider.
+
+For on-premises universes, because the unresponsive node will likely fail to clean up its data directory, home directory, and so on, YBA sets the state to Decommissioned. This prevents the node from being added to a new universe.
+
+### Check on-premises node state
+
+On-premises nodes have three states: In use, Free, and Decommissioned as described in the following illustration.
+
+![Decommissioned node workflow](/images/ee/on-prem-replace-workflow.png)
+To check the state of an on-premises node, navigate to **Configs > Infrastructure > On-Premises Datacenters**, select the associated on-premises configuration, and click **Instances**.
+
+### Recommission a decommissioned on-premises node
+
+You can return a Decommissioned node to the on-premises provider Free pool after it becomes reachable and is cleaned up.
+
+Perform the following steps to recommission a node:
+
+1. Navigate to **Configs > Infrastructure > On-Premises Datacenters**, select the associated on-premises configuration, and click **Instances**.
+
+1. Under Instances, for the decommissioned node, click **Actions > Recommission Node**.
+
+    ![Recommission Node](/images/ee/recommission-node.png)
+
+1. Click OK to confirm.
+
+YugabyteDB Anywhere (YBA) starts the node recommissioning process, and you can view the progress on the **Tasks** tab.
+
 ## Eliminate an unresponsive node
 
 If a virtual machine or a physical server in a universe reaches its end of life and has unrecoverable hardware or other system issues (such as problems with its operating system, disk, and so on) it is detected and displayed in the YugabyteDB Anywhere UI as an unreachable node, as per the following illustration:
 
 ![Unreachable Node Actions](/images/ee/node-actions-unreachable.png)
 
-When this happens, new Master leaders are elected for the underlying data shards. Because the universe enters a partially under-replicated state, it would not be able to tolerate additional failures. To remedy the situation, you can eliminate the unreachable node by taking actions in the following sequence:
+When this happens, new Master leaders are elected for the underlying data shards. Because the universe enters a partially under-replicated state, it would not be able to tolerate additional failures. To remedy the situation, it is recommended that you [replace the node](#replace-a-live-or-unreachable-node).
```

Comment:
```suggestion
When this happens, new tablet leaders are elected for the underlying data shards and availability is maintained. If no other nodes are available to rebuild the lost copies of these tablets, the universe enters a partially under-replicated state and it would not be able to tolerate additional failures. To remedy the situation, it is recommended that you [replace the node](#replace-a-live-or-unreachable-node).
```